server/utility/minerva_utility.py

```diff
@@ -48,6 +48,11 @@ def findMinervaFolder(currentUser, user, create=False):
                            PluginSettings.MINERVA_FOLDER, create)
 
 
+def findPublicFolder(currentUser, user, create=False):
```

Comment:
Oh, I probably unintentionally changed it. If it's not too much trouble, you could change it back.